src/components/TripPlanner/Map.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {useState, useEffect} from "react";
+import {
+  ComposableMap,
+  Geographies,
+  Geography,
+  ZoomableGroup,
+  Marker
+} from "react-simple-maps";
+import { fetchListSighting as fetchAllApprovedSightings, WhaleSighting } from "../../clients/apiClient";
+
+const geoUrl = "https://raw.githubusercontent.com/deldersveld/topojson/master/world-countries.json";
+
+const MapChart = () => {
```

Comment:
We can add the export here so we don't need to add it to the bottom of the file
```suggestion
export const MapChart: React.FC = () => {
```